apps/coa/behaviours/save-desired-content.js

```diff
@@ -0,0 +1,18 @@
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
```

Comment:
use `getValues(req, res, next)` instead of locals. 
locals is used for setting values as key value pairs for rendering the html
getValues gives us a copy of sessionModel so we can get `applicant-full-name` straight away

Reply:
Made changes to getValues behaviour and removed the locals function.